README.md

```diff
@@ -16,3 +26,18 @@ page](https://docs.github.com/en/get-started/writing-on-github/working-with-adva
 might help with the notation for mathematical expressions.
 
 $f(n)\in o(g(n)) \iff \forall c>0, \exists n_0, \forall n\ge n_0: f(n) < c g(n)$
+
+ANSWER:
+
+Definitions:
+little o (given):
+$f(n)\in o(g(n)) \iff \forall c>0, \exists n_0, \forall n\ge n_0: f(n) < c g(n)$
+
+Big O:
+$f(n)\in O(g(n)) \iff \exists c>0, \exists n_0, \forall n\ge n_0: f(n) \leq c g(n)$
+
+Proof: By definition, $f(n)\in o(g(n))$ implies that $f(n)\in O(g(n))$
+
+Using the definitions, we can see that $f(n) < c g(n)$ will be equal to $f(n) \leq c g(n)$ for any c that works with g(n) and f(n) inside little o definition since $f(n) < c g(n)$ is equal to $f(n) \leq c g(n)$ when c, f(n), and g(n) are valid inputs for little o. As such, when $f(n)\in o(g(n))$ is true, $f(n)\in O(g(n))$ is also true. 
```

Comment:
$f(n) < c g(n)$ cannot be equal to $f(n) \leq c g(n)$ -- they are different expressions.